webpack.config.js

```diff
@@ -29,37 +44,42 @@ if (process.env.NODE_ENV !== 'development') {
       },
     },
     module: {
-      loaders: [
```

Comment:
@EdwinGuzman 

If we add `react-dom` to the externals, it reduces the file size from 1.73mb to 1.06 which is huge.

```
      react: {
        root: 'React',
        commonjs2: 'react',
        commonjs: 'react',
        amd: 'react',
        umd: 'react',
      },
      'react-dom': {
        root: 'ReactDOM',
        commonjs2: 'react-dom',
        commonjs: 'react-dom',
        amd: 'react-dom',
        umd: 'react-dom',
      },
```

We could also add `underscore`

Reply:
Good catch. Will do.